### ISSUE TITLE
[Docs] Unify how we call out Enterprise only functionality

### ISSUE THREAD

**Issue by mkeeler** [hashicorp/consul#6887]
The ways we currently show that something is enterprise specific are quite varied and I believe we should fix things to have a common look when calling out the enterprise features.

## On the Agent Config Page

<img width="948" alt="Screen Shot 2019-12-05 at 9 19 15 AM" src="https://user-images.githubusercontent.com/1641976/70243458-9433f400-1740-11ea-8554-998060a53578.png">

## On the ACL Rules Page

<img width="948" alt="Screen Shot 2019-12-05 at 9 18 48 AM" src="https://user-images.githubusercontent.com/1641976/70243465-96964e00-1740-11ea-87e7-7203d70a9afa.png">

## Consul License CLI Docs Page
<img width="948" alt="Screen Shot 2019-12-05 at 9 18 13 AM" src="https://user-images.githubusercontent.com/1641976/70243478-9c8c2f00-1740-11ea-9262-460ed00f0132.png">

## Snapshot Agent Docs Page

<img width="948" alt="Screen Shot 2019-12-05 at 9 17 58 AM" src="https://user-images.githubusercontent.com/1641976/70243551-cb0a0a00-1740-11ea-99cc-14f447e1ae2d.png">

## In Progress Namespace Rules section of the ACL Rules page

<img width="1006" alt="Screen Shot 2019-12-05 at 12 09 40 PM" src="https://user-images.githubusercontent.com/1641976/70257384-2c89a300-1758-11ea-9400-cad2e1ebbd36.png">

**Comment by judithpatudith** [comment]
This is awesome! I vote for the badge because it stands out the most, which can make enterprise users feel special and open source users feel less bait-and-switched. I have a vague memory of seeing it called out as the "right" way in a readme or doc somewhere, although I'm can't find it now 🤔

**Comment by mkeeler** [comment]
For some upcoming docs I also stripped out the black Enterprise box so it could be put next to a sub section in a doc that is mostly applicable to oss & enterprise.

I think for entire pages where all content is for Enterprise only the larger block like we have for the Consul License CLI page is nice and then for everywhere else the small black box.

I had to inline some html to get that small one so we would probably want to pr github.com/hashicorp/middleman in order to make the small one standard somehow.
